packages/firestore/src/core/query.ts

```diff
@@ -635,17 +610,14 @@ export class RelationFilter extends Filter {
   }
 }
 
+export class UnaryFilter extends FieldFilter {}
```

Comment:
I don't care strongly, but this empty class looks a little silly.  I guess it only exists for a single instanceof check in the serializer.  I say we drop it or else add some kind of comment justifying its existence ("for parity with the proto representation" or something)